source/renderer/app/stores/WalletsStore.js

```diff
@@ -44,6 +42,7 @@ import type {
   TransferFundsCalculateFeeRequest,
   TransferFundsRequest,
 } from '../api/wallets/types';
+import { TESTNET_MAGIC } from '../../../main/config';
```

Comment:
@DeeJayElly this breaks the build... You can NOT import anything from the `source/main` into `source/renderer`. Please move the `TESTNET_MAGIC` under `source/common` and then import it in appropriate files in main and renderer...

Reply:
Fixed